rolling-shutter/keyper/fx/send.go

```diff
@@ -42,3 +47,8 @@ func SendShutterMessages(
 		}
 	}
 }
+
+// isRetrieable is a no-op so far.
+func isRetrieable(_ *shmsg.Message) bool {
```

Comment:
I am not sure to be honest. It would be a message, that does not age well. I left that stub in there, to explicitly point out, that currently all messages _will_ be retried. I've added a `FIXME`, so we revisit that later.